src/ontology/cl-edit.owl

```diff
@@ -27439,17 +27441,17 @@ SubClassOf(obo:CL_1001568 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000531
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:pr"^^xsd:string) obo:IAO_0000115 obo:CL_1001569 "An interneuron of the hippocampus."^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1001569 "FMA:84777"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_1001569 "hippocampal interneuron"^^xsd:string)
-EquivalentClasses(obo:CL_1001569 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001954)))
+EquivalentClasses(obo:CL_1001569 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002421)))
```

Comment:
@shawntanzk, I notice some EAs changed in this PR use the 'has soma location' object property while others, like this one, use 'part of'.
Is this intentional?

Reply:
ah thanks for the spot, it should defs be 'has soma location' - will make the change. (guessing this is cause I changed my search and replace initially so missed it)

Reply:
@shawntanzk to fix and merge ASAP